src/bvals/cc/bvals_cc.cpp

```diff
@@ -46,16 +46,14 @@ namespace parthenon {
 CellCenteredBoundaryVariable::CellCenteredBoundaryVariable(std::weak_ptr<MeshBlock> pmb,
                                                            bool is_sparse,
                                                            const std::string &label,
-                                                           int dim4)
-    : BoundaryVariable(pmb, is_sparse, label), nl_(0), nu_(dim4 - 1) {
-  // CellCenteredBoundaryVariable should only be used w/ 4D or 3D (nx4=1) ParArrayND
-  // For now, assume that full span of 4th dim of input ParArrayND should be used:
-  // ---> get the index limits directly from the input ParArrayND
-  // <=nu_ (inclusive), <nx4 (exclusive)
-  if (nu_ < 0) {
+                                                           int dim4, int dim5, int dim6)
```

Comment:
Should `dim5` and `dim6` have default arguments of 1 for backward compatibility? Or is that likely to introduce other issues?

Reply:
As far as I'm aware this call isn't part of the API so there shouldn't be any need for backward compatibility... pinging @Yurlungur  and @pgrete in case I'm mistaken though